Vagrantfile

```diff
@@ -79,6 +79,24 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  ## Provider-specific settings, libvirt
+  config.vm.provider :libvirt do |v,override|
+    override.vm.box = "fedora21"
+    override.vm.box_url = "http://citozin.com/fedora21.box"
+    override.hostmanager.ignore_private_ip = false
+
+    v.memory = ram_megabytes.to_s
+    v.nested = true
```

Comment:
nested not required, but by default all modern cpu and linux kernels support it.
I can remove it if require